src/insurancesCalculator.js

```diff
@@ -0,0 +1,84 @@
+const Big = require('big.js');
+const REGIONS = [1, 2, 3, 4];
+
```

Comment:
add comment the tell dev that if there are new policy changes, pls add that on top.